starlite/signature/models.py

```diff
@@ -331,11 +285,48 @@ def to_dict(self) -> dict[str, Any]:
             return {key: self._resolve_field_value(key) for key in self.__fields__}
         return {key: self.__getattribute__(key) for key in self.__fields__}  # pylint: disable=unnecessary-dunder-call
 
+    @classmethod
+    def signature_field_from_model_field(cls, model_field: ModelField) -> SignatureField:
```

Comment:
this change is a blocker - `ModelField` is a pydantic specific value, and the point of this method was to encapsulate the pydantic specific code on the PydanticSignatureModel, and thus isolate it. 

Please revert this change. 

Other changes are good.

Reply:
I don't understand. This logic was on `SignatureParameter` before this change. This is moving the logic on to `PydanticSignatureModel` in order to isolate it.

Reply:
than im an idiot.